client/connection.go

```diff
@@ -81,7 +81,7 @@ func ConnectLXD(url string, args *ConnectionArgs) (InstanceServer, error) {
 //
 // Unless the remote server is trusted by the system CA, the remote certificate must be provided (TLSServerCert).
 func ConnectLXDWithContext(ctx context.Context, url string, args *ConnectionArgs) (InstanceServer, error) {
-	logger.Debugf("Connecting to a remote LXD over HTTPS")
+	logger.Debug("Connecting to a remote LXD over HTTPS")
```

Comment:
Is there a Go linter that could warn on every `Debugf` call lacking any format specifier?

Reply:
The are quite a few of those actually:

```
$ git grep -E 'logger\.(Debug|Error|Info|Warn)f\(' | grep -cvF '%'
125
```

Reply:
Yeah I just pick up the ones I notice as I go. Otherwise I could spend a long time on code cleanups/standardization.

Reply:
> Is there a Go linter that could warn on every `Debugf` call lacking any format specifier?

Probably, I've not looked.